<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyTwoImageComparePane</class>
 <widget class="QWidget" name="MyTwoImageComparePane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(236,236,236)</string>
  </property>
  <widget class="QLabel" name="m_labelImage1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(180,180,180)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
font: 10pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>相似度</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_labelSimilar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
font: 10pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>99.25%</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="m_progressBar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>118</width>
     <height>10</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid rgb(180,180,180)</string>
   </property>
   <widget class="QLabel" name="m_labelImage2">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>20</y>
      <width>119</width>
      <height>129</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="m_labelTitle">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>119</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color:rgb(255,255,255);
font: 10pt &quot;Arial Unicode MS&quot;;
background-color:rgb(76,76,76)</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>119</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../Resouce.qrc">:/images/libraryLabel.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>m_labelImage2</zorder>
   <zorder>label</zorder>
   <zorder>m_labelTitle</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../../Resouce.qrc"/>
 </resources>
 <connections/>
</ui>
